uber/decorators.py

```diff
@@ -485,3 +485,20 @@ def __call__(self, func):
         redirect_func = create_redirect(self.url or '../' + get_module_name(func) + '/' + func.__name__)
         setattr(root, self.alias_name or func.__name__, redirect_func)
         return func
+
+
+def id_required(func):
+    @wraps(func)
+    def check_id(self, **params):
+        message = "ID NOT FOUND"
```

Comment:
Yeah, the error message for this is tough. I don't think it should say 'you weren't found' because the problem isn't that they're not in the DB. Maybe "No ID provided. Trying using a different link or going back."